src/pretix/base/payment.py

```diff
@@ -278,8 +278,20 @@ def settings_form_fields(self):
                  required=False,
                  disabled=not self.event.settings.invoice_address_required
              )),
+            ('_restrict_to_sales_channels',
+             forms.MultipleChoiceField(
+                 label=_('Restrict to specific sales channels'),
+                 choices=(
+                     (c.identifier, c.verbose_name) for c in get_all_sales_channels().values()
+                     if c.identifier != 'pretixpos'
```

Comment:
Hmm, that's a little special-casey. Do you think we should add a flag to sales channels?